src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -216,7 +216,7 @@ public String getRemoteFs() {
     }
 
     public void setInstanceCap(int instanceCap) {
-        if (instanceCap <= 0) {
+        if (instanceCap < 0) {
```

Comment:
Sounds good.
I thought there might be a situation where the xml is loaded with `instanceCap` not set at all, but if that cannot occur I guess it's fine.